File: R/tests/testthat/test_aversivelearning_gamma.R
context("Test aversivelearning_gamma")
library(hBayesDM)

test_that("Test aversivelearning_gamma", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(aversivelearning_gamma(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_aversivelearning_gamma.py
import pytest

from hbayesdm.models import aversivelearning_gamma


def test_aversivelearning_gamma():
    _ = aversivelearning_gamma(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
